fastboot-server.js

```diff
@@ -6,7 +6,7 @@ if (typeof(PhusionPassenger) !== 'undefined') {
 
 const express = require('express');
 const compression = require('compression');
-const morgan  = require('morgan');
```

Comment:
don't change production code not related to your pull request.

Reply:
I think some of code we're change during the merge. I’ll correct that